components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/models/Application.java

```diff
@@ -195,4 +195,13 @@ public HashMap getPermissionMap() {
     public void setPermissionMap(HashMap permissionMap) {
         this.permissionMap = permissionMap;
     }
+
+    @Override
+    public String toString() {
+        return "Application [name=" + name + ", groupId=" + groupId + ", uuid=" + uuid + ", policyId=" + policyId
```

Comment:
Shouldn't we change it to use StringBuilder if we are using this overridden toString ()method? Or can't we remove it if we don't see a usage?